gatewayd/journal_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gatewayd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+)
+
+func TestGetEntries(t *testing.T) {
+
+	var expected = []*Entry{
+		// currently any fields which systemd prepends with "_" or "__" will be hidden
+		&Entry{
+			CURSOR:                  "s=c60ae6415a0e4531a2064c8cdb11a0f4;i=587;b=d5f050252af342a2817795b375889f78;m=b11f9d6;t=5120f62678669;x=74d4678ad474a6b6",
```

Comment:
That's weird..I must have accidentally broken the tests in a rebase before I submitted the PR. Should be fixed. I also found a bug with something else, noted in a comment to this PR.